backend/src/Model/Bookmark.cs

```diff
@@ -0,0 +1,23 @@
+
+namespace Backend.Model;
+using Postgrest.Attributes;
+using Postgrest.Models;
+
+[Postgrest.Attributes.Table("Bookmark")]
+public class Bookmark:BaseModel{
+    [PrimaryKey("id", false)]
+    public long Id { get; set; }
+
+    [Column("createdAt")]
+    public DateTime CreatedAt { get; set; }
+
+    [Column("imgUrl")]
+    public string? ImgUrl { get; set; }
+
+    [Column("link")]
+    public string? Link { get; set; }
```

Comment:
hmmm program do zapisu linkow z pozycja na link jako opcjonalna xD? Wydaje mi sie ze to pole jednak powinno byc required

Reply:
Racja